modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -570,14 +571,23 @@ describe('TSS Ecdsa Utils:', async function () {
         userKeyShare.nShares[3], backupKeyShare.nShares[3],
       ]);
 
+      const userChallenge = await rangeProof.generateNTilde(3072);
       const bitgoChallenge = await rangeProof.generateNTilde(3072);
-      await nockGetChallenge({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, addendum: '/transactions/0', response: {
-        ntilde: bitgoChallenge.ntilde.toString(),
-        h1: bitgoChallenge.h1.toString(),
-        h2: bitgoChallenge.h2.toString(),
-      } });
 
-      const [userSigningKeyWithChallenge, bitgoSigningKeyWithChallenge] = await Promise.all([MPC.signChallenge(userSigningKey.xShare, userSigningKey.yShares[3]), MPC.signChallenge(bitgoSigningKey.xShare, bitgoSigningKey.yShares[1])]);
+      sinon.stub(commonTssMethods, 'getChallengesForEcdsaSigning').resolves({
+        enterpriseChallenge: {
+          nTilde: userChallenge.ntilde.toString(),
+          h1: userChallenge.ntilde.toString(),
+          h2: userChallenge.ntilde.toString(),
+        },
+        bitGoChallenge: {
```

Comment:
```suggestion
        bitgoChallenge: {
```
This is changed in my PR.